### PR TITLE
Feature/update payex colors

### PR DESCRIPTION
**Pull request by WMeander**
## Description
Updates for PayEx's colors, fonts, logos. Updated Material Icons for PayEx to use "Sharp"

## Motivation and Context
PayEx's design guide is out of date and this is a start to get a working design guide for PayEx.

## How Has This Been Tested?
Changes are mostly done in .less files so no unit tests are added. Existing unit test all pass.
Review has been done via Malin Gutemyr (working with design from PayEx)

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the **CHANGELOG** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: src/less/components/payex/card.less [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            border-bottom: 4px solid @payex-dark-blue;
+        }
+
+        > i {
+            color: @payex-dark-blue;
+        }
+
+        .cards-icon {
+            color: @white;
+            background-color: @payex-dark-blue;
```
Comment (VikingTristan) [inline src/less/components/payex/card.less:34]: Should these colors be rendered out and represented on the "colors" page?

File: src/less/components/payex/progress-step.less [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.steps {
+    ol {
+        li {
+            a {
+                .steps-number {
+                    &:hover {
+                        color: @white;
+                    }
```
Comment (VikingTristan) [inline src/less/components/payex/progress-step.less:8]: Might need to be on li instead

File: src/less/documentation-payex.less [hunk 3]
```diff
@@ -1,3 +1,59 @@
 @import "_variables-swedbankpay.less";
 @import "_variables-payex.less";
 @import "documentation-swedbankpay.less";
+
+body {
+    font-family: @brand-default;
```
Comment (VikingTristan) [inline src/less/documentation-payex.less:6]: doesn't work on some places?

File: src/less/payex.less [hunk 4]
```diff
@@ -2,3 +2,24 @@
...
+@import "components/payex/material-icons.less";
+@import "components/payex/progress-step.less";
+@import "components/payex/radiobutton.less";
+@import "components/payex/sidebar.less";
+@import "components/payex/table.less";
+@import "components/payex/tabs.less";
+@import "components/payex/toast.less";
+@import "components/payex/tooltip.less";
+@import "components/payex/card.less";
+@import "components/payex/help-block.less";
```
Comment (VikingTristan) [inline src/less/payex.less:25]: clean :)

File: src/less/swedbankpay.less [hunk 5]
```diff
@@ -1,5 +1,5 @@
 /* Google fonts */
-@import (css) url("https://fonts.googleapis.com/icon?family=Material+Icons|Material+Icons+Outlined");
+@import (css) url("https://fonts.googleapis.com/icon?family=Material+Icons|Material+Icons+Outlined|Material+Icons+Sharp");
```
Comment (VikingTristan) [inline src/less/swedbankpay.less:2]: remember to remove :)

**Review by VikingTristan — COMMENTED** [review]
Sweet!

A couple of things missing until mergable:
- Font missing on certain paragraphs/content
- Hover effect change to li on progress steps components
- "Colors" page might need a little rework?
- "Swedbank Pay" is still present some places, but this is perhaps not the task of this pull request and can be ignored

File: src/less/documentation-payex.less (hunk 3 above)
Comment (WMeander) [reply src/less/documentation-payex.less:6]: documentation has the right fonts now

File: src/less/components/payex/progress-step.less (hunk 2 above)
Comment (WMeander) [reply src/less/components/payex/progress-step.less]: moved to li but got problems with text under a-tag. Designer from Payex decided to go with hover effects without the color white.

File: src/less/components/payex/card.less (hunk 1 above)
Comment (WMeander) [reply src/less/components/payex/card.less:34]: Colors for payex is added to the identity/colors page, including the dark blue color.

File: src/less/swedbankpay.less (hunk 5 above)
Comment (WMeander) [reply src/less/swedbankpay.less]: removed!

**Review by VikingTristan — APPROVED** [review]
looks good to me 👍 
Anything to add, @PhungNg ?

**Comment by WMeander** [comment]
Sorry! trigger happy, såg inte din kommentar till Phung, @VikingTristan

**Comment by WMeander** [comment]
Ja jag såg att den failade. Ska jag bara restore min branch och köra om jobbet?

**Comment by VikingTristan** [comment]
> Ja jag såg att den failade. Ska jag bara restore min branch och köra om jobbet?

nono its okay ^^

**Pull request merged**